### DISCUSSION TITLE
[REQUEST] Emby media player with h/w decode

### DISCUSSION THREAD

**Discussion by spants** [tteck/Proxmox#445] [Ideas]
Thanks so much for your work. I used to use Plex (lifetime subscriber) but moved across to Emby for the focus on streaming and number of different clients that are available. 
I use EMBY as a docker under Unraid VM but would love to move it out to a Container.

Happy test if you are able to help. (A good starting point may be the jellyfin container)
Thanks again

**Comment by tteck** [comment]
Done!

**Reply by spants** [reply]
will try it shortly, thanks!

**Reply by spants** [reply]
seems to work as a software decoder version but I will need to check out how to do nvidia hardware decoding. I can see that I need to do research! (lxc works - its my knowledge that is lacking!) Thanks again

**Reply by nmbrg** [reply]
ive installed emby using the latest script but no hw-transcode for me. 

![image](https://user-images.githubusercontent.com/31845361/186499027-c2582298-eaf7-4ceb-8d3b-9a02a6746ccd.png)

**Reply by tteck** [reply]
Do you have an active Emby Premiere subscription?

**Reply by nmbrg** [reply]
Yes. 
![image](https://user-images.githubusercontent.com/31845361/186499863-9acf3e91-cd18-4d81-a7d8-e551fb70d6e9.png)

**Reply by nmbrg** [reply]
not sure if this is relevant, but looking at this page it seems a few things needs to be installed or configured.
https://support.emby.media/support/solutions/articles/44001160207-hardware-acceleration-on-linux
(looking at the "required setup steps" for VA API/intel quick sync)

not sure if your script does that?

**Comment by spants** [comment]
I am using a Nvidia card but see the same results. I think some extra configuration is required for hw transcodes

**Comment by LeidenSpain** [comment]
Hi,

I also have a lifetime license, and would be interested in Intel QuickSync Video acceleration.
In docker it was necessary to move the devices (in case it helps):
```
     devices:
       - /dev/dri:/dev/dri
       - /dev/dri/renderD128:/dev/dri/renderD128
       - /dev/dri/card0:/dev/dri/card0

```

**Comment by ghost** [comment]
Similar case here, looking to use this script but want to make sure I can use either my Intel UHD or Nvidia 1650 S for on the fly transcoding

**Reply by LeidenSpain** [reply]
I'm using the UHD630 without problems

**Reply by ghost** [reply]
Would you be able to reproduce the steps you took? Thanks in advance!

**Reply by LeidenSpain** [reply]
Hi,

I have added this line in the LXC configuration file.
This gives access permissions to the graphics card:
`lxc.hook.pre-start: sh -c "chown 0:107 /dev/dri/renderD128"`

Regards
